modules/navier_stokes/src/functormaterials/GeneralFunctorFluidProps.C

```diff
@@ -30,6 +30,10 @@ GeneralFunctorFluidProps::validParams()
       "force_define_density",
       false,
       "Whether to force the definition of a density functor from the fluid properties");
+  params.addParam<bool>(
+      "remove_density_time_derivative_jacobian_contribution",
```

Comment:
there is kind of a forwarding here... as in it s a little bit early to talk about the Jacobian in a functor material.

as you want, we can also have the parameter called 'neglect_density_time_derivative_pT_derivatives'

Reply:
Yeah, rename it like that

Reply:
If specific to p-T derivatives, have parameter description reflect that